docs/source/overview/api.rst

```diff
@@ -175,6 +175,53 @@ Note that a tag is automatically created for the resource with the name ``<names
                 * ``resource_name`` : name of the resource to be created in the Paraglider controller (note: this name will be scoped on cloud and namespace when stored)
                 * ``description``: JSON string describing the resource to be created (excluding networking details)
 
+Attach
+^^^^^^
+
+Attaches a resource to Paraglider according to the resource ID and the specified cloud. 
```

Comment:
Should we make it clearer that the resource should already exist? And if there's any criteria for attaching resources?

Reply:
Added that it should be within a Paraglider deployment. However, didn't want to add that the address spaces shouldn't overlap and permit rules restricitons as that might be too detailed. Thoughts?

Reply:
I think it's ok. I agree it might be a bit verbose, but we should definitely have a requirements list for what you can attach. Otherwise, there's no way for users to know what is attachable.